documentation/docs/configuration/projects.md

```diff
@@ -2,22 +2,42 @@
 sidebar_position: 3
 ---
 
-# Configuration Structure
+# Projects
 
-Configuration files are ordered by `project` in the projects folder. Project folder can only contain:
+A project is a folder containing, which contains specially named
+sub-folders, representing APIs. This API folders then contain another
+layer of folders defining confiugrations. Then finally, this configuration
+folders contain yaml files, specifying what gets deployed.
 
-- configurations
-- or another project(s)
+## APIs
 
-This means, it is possible to group projects into folders.
+To see a list of all supported APIs and folder names, please have a
+look [here](./configTypes_tokenPermissions.md).
 
-Combining projects and configurations in same folder is not supported.
+## Configurations
 
-There is no restriction in the depth of projects tree.
+Configurations consists of two parts:
+- Yaml defining parameters, dependencies, name and template
+- Template file
```

Comment:
JSON Template files

Reply:
We never actually enforce that it has to be a json :laughing: